wagtailio/static/sass/components/_sign-up-form.scss

```diff
@@ -0,0 +1,132 @@
+.sign-up-form {
+    $root: &;
+    @include sf-spacing();
+
+    &__heading {
+        // normalise the h2 element
+        margin: 0;
+    }
+
+    &__sub-heading {
+        padding: 10px 0 0 0;
+
+    }
+
+    &__label {
+        display: none;
+    }
+
+    &__icon {
+        display: none;
+
+        @include media-query(medium) {
+            display: inherit;
+        }
+
+        > svg {
+            inset: -5% 3% auto auto;
+            width: 170px;
+            height: 253px;
+
+            @include media-query(large) {
+                inset: 10% 86px auto auto;
+            }
+        }
+    }
+    
+    &__container {
+        display: flex;
+        flex-direction: column;
+        
+        @include media-query(medium) {
+            flex-direction: row;
+        }
+
+        > fieldset {
+            display: flex;
+            flex-basis: start;
+            @include media-query(medium) {
+                width: 50%;
+            }
+        }
+    }
+
+    &__input {
+        flex: 1;
+        padding: 11px 20px;
+        height: 50px;
+        border: 1px solid $color--off-black;
+        flex-grow: 5;
```

Comment:
Suggestion: remove this line as you have `flex: 1` on line 55 (which is shorthand for flex-grow)

```
flex: 1;
```
is equal to:
```
flex-grow: 1;
flex-shrink: 1;
flex-basis: 0%;
```